ecosystem/sep-0010.md

```diff
@@ -124,6 +131,9 @@ On success the endpoint must return `200 OK` HTTP status code and a JSON object
       * `manage_data(source: server account, key: 'web_auth_domain', value: web_auth_domain)`
         * The source account is the **Server Account**
         * The value is the **Server**'s domain. It can be at most 64 characters.
+      * (optional) `manage_data(source: client domain account, key: 'client_domain', value: client_domain)`
+        * The source account is the **Client Domain Account**
+        * Only add this operation if the client passed a `client_domain` parameter in the request and the server supports [Verifying the Client Domain](#verifying-the-client-domain)
```

Comment:
💡 I think it is worth reordering these to first gate including the operation on what the server supports. I'm a little concerned in a quick read someone might think they should include it only based on what is in the request.
```suggestion
        * Add this operation if the server supports [Verifying the Client Domain](#verifying-the-client-domain) and the client provided a `client_domain` parameter in the request.
```